### ISSUE TITLE
Pull requests for next release?

### ISSUE THREAD

**Issue by carestad** [nathanreyes/v-calendar#288]
Do you accept pull requests for the next branch?

My PR for adding support for Norwegian locale (#191) has been lying around for quite some time, and I would love it if we could at least get it into next (v1). It seems the `locales.js` file has been moved into a `defaults` folder, but that should be it.

I will be happy to do the work with the PR (not a lot really), but would like to know if you accept any, since no PRs have been merged since march 2018.

**Comment by nathanreyes** [comment]
Yes I'll be sure to take a look. Apologies for not getting it in sooner. Been really scrambling to get this out of beta.
